docker-compose.yml

```diff
@@ -7,7 +7,7 @@ networks:
 services:
   localtest_loadbalancer:
     container_name: localtest-loadbalancer
-    image: localtest-loadbalancer:latest
+    image: nginx:alpine-perl
```

Comment:
Removing the version number here would pull the latest one, but at the same time it breaks renovate, right? I think we should keep the version number and let renovate do its job (especially if we add tests in the future that make sure the containers can run properly).

Reply:
Nginx has been brutally stable... never had any reason to lock docker:image version. It's slightly more performant to target "latest" in our current setup, but we could easily target a spesific image. It really doesn't matter.